backend/src/main/java/org/eclipse/tractusx/puris/backend/masterdata/controller/MaterialController.java

```diff
@@ -55,6 +56,7 @@ public class MaterialController {
         @ApiResponse(responseCode = "409", description = "Material with the given ownMaterialNumber already exists."),
         @ApiResponse(responseCode = "500", description = "Internal Server error.")
     })
+    @CrossOrigin
     public ResponseEntity<?> createMaterial(@RequestBody MaterialEntityDto materialDto) {
```

Comment:
Did you not add a check here by intention? I initially thought about that you dind't add validations as this is a mad interface and therefore commonly not related to user input. Then I saw, that this is done for the material-partner-relationship. Thus, it would make sense to add the validation for the create and update material methods, too. As we're using here the dtos, the pattern check can be added there.